WalletWasabi.Gui/Controls/WalletExplorer/CoinJoinTabViewModel.cs

```diff
@@ -39,7 +39,7 @@ public class CoinJoinTabViewModel : WalletActionViewModel
 		private const string EnqueuingButtonTextString = "Queuing coins...";
 		private string _dequeueButtonText;
 		private const string DequeueButtonTextString = "Dequeue Selected Coins";
-		private const string DequeuingButtonTextString = "Dequeuing coins...";
+		private const string DequeueingButtonTextString = "Dequeueing coins...";
```

Comment:
NACK, the spelling was correct. Also I don't see the change in the xaml, so I assume this breaks the CoinJoin tab.

Reply:
Actually, [dequeueing](https://www.thefreedictionary.com/dequeueing) is correct.